libnopegl/src/node_rtt.c

```diff
@@ -79,6 +79,7 @@ static const struct node_param rtt_params[] = {
 };
 
 struct rtt_texture_info {
+    struct ngl_node *node;
```

Comment:
Can be const?

Reply:
Const !

Reply:
Sorry I forgot to push. But, while it can be made const, we modify this priv data. So I think it is better to keep it that way.